SignalFx.LambdaWrapper/AspNetCoreServer/Extensions.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Amazon.Lambda.APIGatewayEvents;
+using Amazon.Lambda.Core;
+using com.signalfuse.metrics.protobuf;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using Newtonsoft.Json;
+
+namespace SignalFx.LambdaWrapper.AspNetCoreServer
+{
+    public static class Extensions
+    {
+        private const string WrapperVersion = "0.1.0";
```

Comment:
This wrapper version and the wrapper version in `MetricWrapper.cs` are inconsistent. Let's bump them up and make them the same. Maybe should think about moving this out to a config file as well since they are a shared value.